shell/platform/tizen/external_texture_pixel_gl_stub.cc

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/tizen/external_texture_pixel_gl.h"
+
+#ifdef TIZEN_RENDERER_EVAS_GL
+#undef EFL_BETA_API_SUPPORT
+#include "tizen_evas_gl_helper.h"
```

Comment:
The only difference between ```external_texture_pixel_gl.cc``` and ```external_texture_pixel_gl_stub.cc``` is this header file. Do we really need ```external_texture_pixel_gl_stub.cc```?

Reply:
I put this part because I thought it could be used when implementing the externel texture of the desktop later.
Right now we don't need it. It would be better to just remove it at this moment.

Reply:
:scissors:  done!